core/src/test/java/org/infinispan/functional/AbstractFunctionalTest.java

```diff
@@ -68,8 +68,7 @@ protected void createCacheManagers() throws Throwable {
 
    protected void configureCache(ConfigurationBuilder builder) {
       // Make sure there are no errors when counting stats
-//      builder.jmxStatistics().enabled(true);
-      builder.jmxStatistics().available(false);
+      builder.statistics().available(false);
```

Comment:
That comment is definitely wrong now.
Funny enough, the commit disabling stats says "ISPN-8651 Add statistics for functional commands" :)

Reply:
If Radim did indeed add that support then the comment should go away and also builder.statistics().available(false) should go. Making stats unavailable was originally meant to remove the CacheMgmtInterceptor from the chain because it did not play well with functional commands. I remember CCE and great fire. But this supposedly works in 2020. Let me try to make them enabled and have a local run.

Reply:
The `available` attribute was actually added in `ISPN-5542 Shorten interceptor stack for local caches`, and if you read the JIRA comments it's obvious that it was motivated by Radim's Hibernate 2LC work.

Sorry if I mislead you, it just seemed funny to me that a commit adding support for statistics would actually change a test to disable statistics. And I'm pretty sure the ClassCastExceptions started appearing *after* ISPN-8651 went in :)

Reply:
Enabling stats there seems to work now :)))

Reply:
That's good to know, I don't see any tests specifically looking at stats in functional maps.